Fixed typos and restored the semantic forms

h8nor opened the pull request:
+ BOM was deleted at the files with encoding of UTF-8

h8nor commented:
Fixed #13 in lines 643-645 of the file "words.html"

toby3d commented on words.html:
```diff
@@ -718,14 +717,14 @@
 
 <p id="character_random">
 <b>СЛУЧАЙНЫЙ:</b><br>
-<i>Обезьяний робот! Ниндзя тако пицца! Лол я такой непредсказуемый</i><br>
```
Думаю здесь всё-таки лучше оставить `Лол`, как насмешку персонажа над самим собой.

h8nor replied:
Вернул как было.

toby3d commented:
Спасибо за PR!
Пожалуйста, убедитесь что весь текст всё ещё умещается в предназначеном для него пространствах. Если это не так - сократите проблемные блоки текста.

toby3d commented on notes/index.html:
```diff
@@ -38,7 +38,7 @@ <h3>"Игра на доверие"</h3>
 <h3>"Ты не будешь знать заранее, когда последний раунд"</h3>
 
-<p>В повторяющейся игре на доверие (также известной как <em>Итерационной Дилемме Заключённого</em>) важно, чтобы ни один из игроков не знал, <em>когда</em> последний раунд. Зачем? Подумайте об этом - в последнем раунде, оба игрока будут знать, что их действие не имеет никакого значения, поэтому они оба сжульничают. Но это означает, что во предпоследнем раунде их действия не могут изменить следующ ...
+<p>В повторяющейся игре на доверие (также известной как <em>Итерационной Дилемме Заключённого</em>) важно, чтобы ни один из игроков не знал, <em>когда</em> последний раунд. Зачем? Подумайте об этом - в последнем раунде, оба игрока будут знать, что их действие не имеет никакого значения, поэтому они оба сжульничают. Но это означает, что в предпоследнем раунде их действия не могут изменить следующи ...
```
Не совсем ясно в чём заключаются изменения. :thinking:

h8nor replied:
Частица "во" была не к месту. Исправил на "в предпоследнем раунде".

toby3d replied:
Да, вижу. Вопрос снят.

toby3d commented on words.html:
```diff
@@ -804,6 +803,7 @@
 
 <p id="noise_evo_5">
 <b>Используй ползунок внизу, чтобы изменить вероятность "недопонимания", затем нажми СТАРТ.</b>
+<br><br>
```
Есть вероятность что эти переносы сломают визуальное представление текста, например заставив уехать за другие элементы. Надо проверить.

h8nor replied:
Я проверил: не сломают. У блока достаточно ещё места.

Minoru replied:
Подтвержаю: там в левой части экрана большая картинка, а справа совсем чуток текста. Парочка переводов строк ничего не сломают.

h8nor commented:
Так, похоже я не удалил BOM, а насорил ими. :smirk_cat: Придётся сделать ещё 2 коммита.

Commit:
Trying to get rid of BOM through the iconv filters

Minoru commented on words.html:
```diff
@@ -643,7 +642,7 @@
 Происходит то же самое:
 с низкой "беспроигрышной" наградой, <span class="all_d">Обманщики</span> берут верх.
-В теории игр на этот счёт есть две сильные идеи:
+Теория игр на этот счёт имеет две идеи:
```
Мне кажется, по-русски так не говорят. Читается, как калька с английского. Можно пояснить мотивацию этого изменения?

h8nor replied:
В данном случае мотивацией является только читаемость текста под кнопкой #13 на странице.

Minoru replied:
Не понимаю, о какой кнопке речь. Вот что я вижу на этой страничке: 
![screenshot-2018-4-9](https://user-images.githubusercontent.com/118875/38515070-c05179cc-3c3b-11e8-9dc5-3ac3adc4095e.png)

Какая кнопка имеется в виду?

h8nor replied:
1. https://screenshots.firefox.com/aFLXoIMOo67qJZNX/notdotteam.github.io
2. https://screenshots.firefox.com/UABAmrVX6hpRtqow/notdotteam.github.io

Minoru replied:
М, понятно. Я бы просто убрал слово «сильные». @toby3d, что думаешь на этот счёт?

toby3d replied:
@Minoru Соглашусь. Думаю мы не потеряем много смысла отбросив одно слово.

Minoru commented on words.html:
```diff
@@ -718,9 +717,9 @@
-если ты обманывать, я делать <i>свой предыдущий ход наоборот</i>, даже если это ошибка.
+Привет! На первом ходу я доверюсь.
+Если ты тоже доверишься, я повторю <i>свой предыдущий ход</i>, даже если твой ход, вероятно, был ошибкой.
+Если ты обманешь, я сжульничаю <i>в свой следующий ход</i>, даже если твой ход, вероятно, был ошибкой.
```
Примитизированный язык здесь не зря; таким образом обыгрывается название персонажа. В оригинале так же. Считаю, что это изменение нужно откатить.

h8nor replied:
Возможно, данный диалект является частью игры. Но чтобы понять правила, которым подчиняется данный персонаж, мне потребовалось несколько итераций. Могу изменить текст под диалект, например, реднеков (Васьков), гопников, школьников или алкоголиков.
> привет. я отдам тебе монету.
> если ты отдаш мне монету то и я отдам тебе монету в следующий раз. даже если это ошибка.
> если ты обманеш меня то в следующий раз я монету не отдам. даже если это ошибка.

Устраивает или будете голосовать?

Minoru replied:
Речь школьника или даже ребёнка здесь была бы к месту, можно попробовать. (Но я не представляю, как это будет выглядеть — т.е. сам написать не могу :)

h8nor replied:
"Двоечники" пишут с ошибками в элементарных правилах без знаков препинания :)

toby3d replied:
Персонаж - "простак". То есть надо обыграть его примитивное мышление и очень упрощённую речь. Либо это чувак с деревни, либо двоечник или что-то вроде.

Minoru replied:
GitHub не присылает уведомления о правке комментариев, так что твои правки выше не сразу бросились в глаза; извини.

Мне не нравится подмена смысла в «я отдам тебе монету» и «даже если это ошибка»:

1. игроки отдают монеты какой-то машине, а не друг другу;
2. в исходнике речь шла про ошибку противника, а в предлагаемой версии речь уже об ошибке Простака.

Также не уверен, что ошибка всего лишь в двух словах достаточно хорошо передаёт смысл персонажа.

@bopoh13, а ты читал англоязычный исходник? Может, удастся оттуда почерпнуть вдохновения?

h8nor replied:
Да, согласен. Присутствует двусмысленность. Оригинал не читал; попробую составить другой вариант.

h8nor replied:
У детей в устной речи возникают ошибки, которые перетекают в дисграфию. Неправильное спряжение глаголов не самый частый вариант, но и не самый путанный. Добавлю грамматических ошибок. Заменить фразу "если это ошибка" не могу.
> превет. сначала я доверять тебе.
если ты тоже доверять мене, я повторять <i>свой ход</i>, даже если это ошибка.
если ты обмануть меня, я делать <i>свой ход наоборот</i>, даже если это ошибка.

toby3d replied:
Может заменить `даже если __это__ ошибка` на `даже если __была__ ошибка`? :thinking: 

**P.S.:** стольким графоманством занимаемся, хоспаде. х)

Minoru replied:
Хм, а ведь в оригинале было так:

> if you cooperate back, i do *same thing* as last move, even if it mistake.

То есть непонятно, про текущий ход речь или про предыдущий, но зато совершенно ясно, что Простак говорит о *своём* ходе, а не ходе противника. Я выше был неправ, пардон.

Мне нравится последний вариант, предложенный @bopoh13 в комментарии. Предлагаю оставить его.

Commit:
Final edits after unanimous vote
+ replaced Unicode chars with mnemonic/numeric code

Commit:
Deleted .gitattributes file
i/lf    w/crlf  attr/text=auto eol=crlf README.md
i/lf    w/crlf  attr/text=auto eol=crlf notes/index.html
i/lf    w/crlf  attr/text=auto eol=crlf notes/notes.md
i/lf    w/crlf  attr/text=auto eol=crlf words.html

h8nor commented:
Проверьте, пожалуйста, перенос строк в файле 'crlf' или 'lf"? Много изменений получилось.

toby3d commented:
@bopoh13 Смотрю последние изменения и не понял почему теперь каждый блок засвечен как подвергнувшийся правкам.

ИМХО: Переносы строк между блоками не важны. Они скорее для читаемости исходного кода.

h8nor commented:
@toby3d, всё-таки в _*nix_-ах и _[Windows](https://stackoverflow.com/questions/3206843/)_ разные символы переноса строк. Избавится от пробелов при просмотре можно добавив `?w=1`, а вот [о символах переноса строки](https://help.github.com/articles/dealing-with-line-endings/) ничего не нашёл.
У GitHub файлы должны храниться как в _*nix_-ах. Похоже, мои фильтры работают не корректно.

Commit:
Changed the line break character to CL
after replacing 'git config --global core.autocrlf false'

Minoru commented on words.html:
```diff
@@ -184,7 +184,7 @@
 
 <p id="oneoff_2_cooperated">
-Ну да, это кажется правильным... <b>ИЛИ НЕТ??</b>
+Ну да, это кажется правильным... <b>ИЛИ НЕТ!?</b>
```
[В оригинале](https://github.com/ncase/trust/blob/1cee99ea9800cc7f1084ff50efddcc527f83107a/words.html#L188) два вопросительных знака:

> Sure, seems like the right thing to do... <b>OR IS IT??</b>

BopoH13 committed:
Author's spelling

Minoru commented:
LGTM. Спасибо!

h8nor commented:
Спасибо. Я тоже рад, что мы договорились )

h8nor commented:
**[Заметка]** Причины нежелательного изменения строк и методы устранения ошибки:
1. Все блоки в файле - стандартный текстовый редактор Windows перед символом-LF конца строки ставит символ-CR возврата каретки.  
= Можно удалить перед индексацией:
``` bash
find . -type f \( -name "*.html" -o -name "*.js" -o -name "*.liquid" -o -name "*.md" \) -print0\
 | xargs -0 grep -m1 -l `printf "^\xEF\xBB\xBF"` | xargs sed -i "1 s/^\xEF\xBB\xBF//; s/\$/\x0D/"\
 && git ls-files -mo --eol
```
2. Отдельные не англоязычные блоки - изменилась кодировка файла.  
= Следует проверить фильтры в `.gitattributes`. Можно ыывести `git ls-files --eol` окончания строк в файлах репозитория (слева) и на локальной машине (справа).

Minoru commented:
По первому пункту у Git уже есть готовая настроечка — core.autocrlf. По умолчанию false, что значит «не конвертировать окончания строк», но на Windows народ часто переключает в true:
```
$ git config core.autocrlf false
```

Выполнять нужно в конкретном репозитории. Если добавить флаг `--global`, эта настройка будет применятся вообще во всех твоих репозиториях.

Настройка конвертирует LF в CRLF при checkout и обратно при commit, т.е. в репозитории будут LF-окончания, а в твоей рабочей копии — CRLF.

bopoh13 commented:
@Minoru, к сожалению, данная настройка [не убирает BOM из файлов в UTF-8](https://github.com/desktop/desktop/issues/3768) после редактирования мной в Notepad (поэтому пришлось экспериментировать, чтобы не ломать исходную кодировку).

bopoh13 commented:
@Minoru, к сожалению, данная настройка сама по себе [не убирает BOM из файлов в UTF-8](https://github.com/desktop/desktop/issues/3768) после редактирования мной в Notepad (поэтому пришлось экспериментировать, чтобы не ломать исходную кодировку).

bopoh13 commented:
@Minoru, к сожалению, данная настройка сама по себе [не убирает BOM из файлов в UTF-8](https://github.com/desktop/desktop/issues/3768) после редактирования мной в Notepad (поэтому пришлось экспериментировать, чтобы не дописывать мусор в виде символом BOM или не ломать исходную кодировку).

h8nor commented:
@Minoru, в целом вы правы - настройка необходима. Но данная настройка после редактирования _мной_ текста в Notepad [не убирает BOM из файлов в UTF-8](https://github.com/desktop/desktop/issues/3768) (рекомендую пользователям Windows перед индексацией применять команду, чтобы не дописывать в файлы "мусор" в виде символов BOM).

toby3d commented:
Самый действенный совет это, конечно, не использовать Notepad вовсе и найти, наконец, нормальный текстовой редактор.

**P.S.:** Только не Word и не Wordpad, боже упаси...

h8nor commented:
https://github.com/NotDotTeam/trust/blob/4a99513ae46b72b9747812fb7e62535a09e480db/words.html#L593-L594
Джон Стюарт Милль воспринимал Иммануила Канта именно так (как последствия), но на мой взгляд это проверка на отсутствия психопата: у которого на первом плане всегда стоят личные желания и нужды, когда другой человек является средством (без уважения как к разумному существу, без признания достоинств, с манипуляциями и желанием управлять), а не целью.
